corehq/apps/hqadmin/templates/hqadmin/email/superuser_staff_email.html

```diff
@@ -1,22 +1,22 @@
-<p>The following users had changes to their superuser and staff statuses (changes in RED):</p>
+<p>The following users had changes to their superuser and staff statuses:</p>
 
 <table>
   <tbody>
     <tr>
-      <td>Users</td>
-      <td>is Superuser</td>
-      <td>is Staff</td>
+      <td><u>Users</u></td>
+      <td><u>is Staff</u></td>
+      <td><u>is Superuser</u></td>
     </tr>
     {% for user in user_changes %}
       <tr>
         <td>{{ user.email }}</td>
         {% if 'is_staff' in user %}
-          <td style="color:red">{{ user.is_staff }}</td>
+          <td><s>{{ user.is_staff|yesno:"False,True" }}</s> -> <b>{{ user.is_staff }}</b></td>
```

Comment:
Nit: This probably works fine, but literal `>` in html is iffy. Since `-&gt;` is unreadable, I might suggest `{{ '->' }}` which I believe renders to `-&gt;` (which html renders to ->). Though, now that I say that you can always just use the unicode right arrow character: '→'.